core/domain/prod_validation_jobs_one_off.py

```diff
@@ -1098,6 +1098,17 @@ def _get_external_id_relationships(cls, item):
         }
 
 
+class CollectionRightsAllUsersModelValidator(BaseModelValidator):
+    """Class for validating CollectionRightsAllUsersModel."""
+
+    @classmethod
+    def _get_external_id_relationships(cls, item):
+        return {
+            'collection_rights_ids': (
+                collection_models.CollectionRightsModel, [item.id])
```

Comment:
Add a mapping in `CollectionModelValidator` as well i.e for each `CollectionModel`, we should have a `CollectionRightsAllUsersModel`.

Reply:
Done.